### PR TITLE
community.mozilla.org

### PR DESCRIPTION
**Pull request by piquark6046**
### To avoid invalid pull requests, please check and confirm following terms

- [X] This is not an ad/bug report;
- [X] My code follows the [guidelines](https://github.com/AdguardTeam/AdguardFilters/blob/master/CONTRIBUTING.md) and [syntax](https://kb.adguard.com/general/how-to-create-your-own-ad-filters) of this project;
- [X] I have performed a self-review of my own changes;
- [X] My changes do not break web sites, apps and files structure.

### If the problem does not fall under any category that is listed here, please write a comment below in corresponding section

- [ ] Missed ads or ad leftovers;
- [X] Website or app doesn't work properly;
- [ ] AdGuard gets detected on a website;
- [ ] Missed analytics or tracker;
- [X] Social media buttons — share, like, tweet, etc;
- [ ] Annoyances — pop-ups, cookie warnings, etc;
- [ ] Filters maintenance.

### Enter the issue address

 - https://github.com/AdguardTeam/AdguardFilters/issues/138101

### Add your comment and screenshots

None.

### Terms

- [X] By submitting this issue, I agree that pull request does not contain private info and all conditions are met

### CODE REVIEW

File: AnnoyancesFilter/Cookies/sections/cookies_allowlist.txt [hunk 1]
```diff
@@ -8,6 +8,10 @@
...
 @@||cdn.cookielaw.org/scripttemplates/otSDKStub.js$domain=nintendo.*|mtv.com|ryobitools.eu|cc.com|3sat.de|zdf.de|costco.com|giantbomb.com|glovoapp.com|rte.ie|digitaltrends.com|spiele.bild.de|spiele.stern.de|vanityfair.com|saarbruecker-zeitung.de|olympics.com|tvn.pl|bergfreunde.de|oe24.at|epicgames.com|toteraz.pl|natgeotv.com|midomi.com|volksfreund.de|kawasaki.com|foxchannels.com.tr|ga.de|shop.mango.com|kralmuzik.com.tr|westernunion.com|roastmarket.de|new.cappex.com|vimeo.com|heute.at|ntv.com.tr|wargaming.net|metacritic.com|gq-magazine.co.uk|viafree.dk|viafree.fi|viafree.no|viafree.se|glamour.de|cbsnews.com|worldoftanks.ru|vg247.com|newyorker.com|games.miamiherald.com|eurogamer.es|eurogamer.de|bet.com|ikonpass.com|mcdelivery.de|id.trimble.com|skateboarding.transworld.net|newegg.com|docs.vmware.com|udemy.com|olx.pl|usmagazine.com|uefa.com|milwaukeetool.eu|muscleandfitness.com|20min.ch|ted.com|elisa.fi|nhl.com
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/138101
+@@||community.mozilla.org/wp-content/themes/mozilla/js/gdpr.js
+||community.mozilla.org/wp-content/themes/mozilla/js/gdpr.js$redirect=noopjs,important
+@@||community.mozilla.org/wp-content/uploads/$image
```
Comment (zloyden) [inline AnnoyancesFilter/Cookies/sections/cookies_allowlist.txt:14]: Does it happens everywhere or only on `community.mozilla.org`?

File: AnnoyancesFilter/Cookies/sections/cookies_allowlist.txt [hunk 2]
```diff
@@ -8,6 +8,10 @@
 ! @@||cdn.cookielaw.org/scripttemplates/otSDKStub.js
 @@||cdn.cookielaw.org/scripttemplates/otSDKStub.js$domain=nintendo.*|mtv.com|ryobitools.eu|cc.com|3sat.de|zdf.de|costco.com|giantbomb.com|glovoapp.com|rte.ie|digitaltrends.com|spiele.bild.de|spiele.stern.de|vanityfair.com|saarbruecker-zeitung.de|olympics.com|tvn.pl|bergfreunde.de|oe24.at|epicgames.com|toteraz.pl|natgeotv.com|midomi.com|volksfreund.de|kawasaki.com|foxchannels.com.tr|ga.de|shop.mango.com|kralmuzik.com.tr|westernunion.com|roastmarket.de|new.cappex.com|vimeo.com|heute.at|ntv.com.tr|wargaming.net|metacritic.com|gq-magazine.co.uk|viafree.dk|viafree.fi|viafree.no|viafree.se|glamour.de|cbsnews.com|worldoftanks.ru|vg247.com|newyorker.com|games.miamiherald.com|eurogamer.es|eurogamer.de|bet.com|ikonpass.com|mcdelivery.de|id.trimble.com|skateboarding.transworld.net|newegg.com|docs.vmware.com|udemy.com|olx.pl|usmagazine.com|uefa.com|milwaukeetool.eu|muscleandfitness.com|20min.ch|ted.com|elisa.fi|nhl.com
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/138101
+@@||community.mozilla.org/wp-content/themes/mozilla/js/gdpr.js
+||community.mozilla.org/wp-content/themes/mozilla/js/gdpr.js$redirect=noopjs,important
```
Comment (zloyden) [inline AnnoyancesFilter/Cookies/sections/cookies_allowlist.txt:13]: As I recall, you should add the type of the request before the redirect modifier like `$script,redirect=noopjs,important`, but for what reason do you need the rule?

Comment (piquark6046) [reply AnnoyancesFilter/Cookies/sections/cookies_allowlist.txt:13]: Blocking `/gdpr.js` without a redirection rule interrupts making new requests related to the images.

File: AnnoyancesFilter/Cookies/sections/cookies_allowlist.txt (hunk 1 above)
Comment (piquark6046) [reply AnnoyancesFilter/Cookies/sections/cookies_allowlist.txt:14]: As far as i know, it happens only on the domain.

Comment (zloyden) [reply AnnoyancesFilter/Cookies/sections/cookies_allowlist.txt:14]: In that case, please limit the rule to this site using `$domain=community.mozilla.org` and check how it works on the app.

Comment (piquark6046) [reply AnnoyancesFilter/Cookies/sections/cookies_allowlist.txt:14]: But, does it not enough to specify the domain by existing `community.mozilla.org` domain in the current rules?

File: AnnoyancesFilter/Cookies/sections/cookies_allowlist.txt (hunk 2 above)
Comment (zloyden) [reply AnnoyancesFilter/Cookies/sections/cookies_allowlist.txt]: Hm, what kind of images? Those two works fine only with exception of the images.

File: AnnoyancesFilter/Cookies/sections/cookies_allowlist.txt (hunk 1 above)
Comment (zloyden) [reply AnnoyancesFilter/Cookies/sections/cookies_allowlist.txt:14]: Looks like it's used only there, so yep, not necessary.

File: AnnoyancesFilter/Cookies/sections/cookies_allowlist.txt (hunk 2 above)
Comment (piquark6046) [reply AnnoyancesFilter/Cookies/sections/cookies_allowlist.txt]: Ah.. It was a mistake.

**Review by zloyden — APPROVED** [review]

**Pull request merged**